README.md

```diff
@@ -16,6 +16,71 @@ In addition to these requirements, we also expect projects to use source control
 
 The full checklist can be seen here: [`PRODUCTIONISATION.md`][link_productionisation].
 
+## Why does this list exists?
+
+We believe that following this list will encourage teams to consistently follow best practices and improve their projects, which would lead to making a greater impact by delivering faster and learning more.
+
+## How to apply this to your project
+  1. Copy the check-list up for your project - in a new tab in the [spreadsheet][spreadsheet].
+  2. Continuously update the checklist, at least once a month.
+  3. Provide feedback, addition and removal of points to the Chalet.
+
+### Granularity
+  Each project that has a unique set of staff/team members, instances and stack, should have its own checklist.
+
+### Definition of Done
+  Once you’re comfortable that you have satisfied the criteria, tick the box. If you’re on the fence, don’t.
+
+## How to measure success
+We measure success by considering the following points:
+
+### Improving the reliability of applications for users
+
+#### Outcomes
+  - All teams tracking positive progress on checklist
+  - Low level of bugs being reported
+  - Customers perceive our applications to be reliable
+  - User perceive our applications to be reliable
+
+#### When will we review
+  - Monthly p15n check-in during Learn Tech before quarterly update
+
+#### What do we want to see
+  - See >5% improvement per month per application
+  - Positive feedback regarding low level of bugs when interviewing delivery managers
+  - Positive review from customer from delivery managers
+
+### Improving the reliability of applications for users
```

Comment:
Duplication of lines `37-51` in lines `53-67`. 😮

Reply:
🤦‍♀